descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "inventory",
-      "version": "9.4",
+      "version": "9.5",
```

Comment:
There doesn't appear to be any changes to the JSON schema, should these new properties be added?

Reply:
Sure.